### PR TITLE
6 fix hard coded resolution

### LINKED ISSUE #6: Fix hard coded resolution
https://github.com/catie-aq/zephyr_ilitek-ili9163c/blob/e0a079b94a8c95bc88bcfee71749982e516504ee/drivers/display/ilitek/ili9163c/ili9163c.c#L443

### PR DESCRIPTION
**Pull request by Sokario**
Get device property value `width` and `height` instead of hard coded defines.

### CODE REVIEW

File: drivers/display/ilitek/ili9163c/ili9163c.h [hunk 1]
```diff
@@ -70,9 +70,9 @@
 #define ILI9163C_GAMADJ_LEN   1U
 #define ILI9163C_MADCTL_LEN   1U
 
-/** X resolution (pixels). */
+/** SCREEN X resolution (pixels). */
 #define ILI9163C_X_RES 128U
-/** Y resolution (pixels). */
+/** SCREEN Y resolution (pixels). */
 #define ILI9163C_Y_RES 160U
```
Comment (clementlgl) [inline drivers/display/ilitek/ili9163c/ili9163c.h:76]: I think these lines can be removed 🤔

File: drivers/display/ilitek/ili9163c/ili9163c.c [hunk 2]
```diff
@@ -440,8 +440,8 @@ static const struct ili9163c_quirks ili9163c_quirks = {
 			},                                                                         \
 		.pixel_format = DT_INST_PROP(n, pixel_format),                                     \
 		.rotation = DT_INST_PROP(n, rotation),                                             \
-		.x_resolution = ILI9163C_X_RES,                                                    \
-		.y_resolution = ILI9163C_Y_RES,                                                    \
+		.x_resolution = DT_INST_PROP(n, width),                                                    \
+		.y_resolution = DT_INST_PROP(n, height),                                                    \
```
Comment (clementlgl) [inline drivers/display/ilitek/ili9163c/ili9163c.c]: Please line up the `\`

**Review by clementlgl — CHANGES_REQUESTED** [review]

File: drivers/display/ilitek/ili9163c/ili9163c.c (hunk 2 above)
Comment (Sokario) [reply drivers/display/ilitek/ili9163c/ili9163c.c]: Resolved in [0c6afee](https://github.com/catie-aq/zephyr_ilitek-ili9163c/pull/8/commits/0c6afee60a897ead16390962a16dc220d7aa0dbe).

File: drivers/display/ilitek/ili9163c/ili9163c.h (hunk 1 above)
Comment (Sokario) [reply drivers/display/ilitek/ili9163c/ili9163c.h:76]: No might be usefull for set_rotation

**Review by clementlgl — APPROVED** [review]

**Pull request merged**